### ISSUE TITLE
About the results of the MTD dataset

### ISSUE THREAD

**Issue by ShileiZeng** [TArdelean/BlindLCA#4]
Could you please share how the training and test sets are split? When I used the same split method as described in the paper of Anomaly Clustering, the metrics I obtained were lower than those reported in the paper.

**Comment by TArdelean** [comment]
Hi,
Our method is fully unsupervised (Blind Anomaly Localization), therefore there is no training set to speak of. We use the entire test set from MTD for evaluation.
Note that due to randomness in the fitting process you may still get slightly different results compared to those reported in the paper.
